charts/mod-invoice-storage/values.yaml

```diff
@@ -164,6 +164,12 @@ integrations:
     querytimeout: 60000
     maxpoolsize: 5
     existingSecret: ""
+  
+  kafka:
+    enabled: true
```

Comment:
Please set enabled: false by default.

Reply:
++

Reply:
@epam-avramenko
Adjusted accordingly to your recommendation.